class.jetpack.php

```diff
@@ -7451,6 +7451,24 @@ public static function get_products_for_purchase() {
 			'included_in_plans' => array( 'search-plan' ),
 		);
 
+		$products[] = array(
+			'key'               => 'anti-spam',
+			'title'             => __( 'Jetpack Anti-Spam', 'jetpack' ),
+			'short_description' => __( 'Automatically clear spam from comments and forms.', 'jetpack' ),
+			'learn_more'        => __( 'Learn More', 'jetpack' ),
+			'description'       => __( 'Automatically clear spam from comments and forms. Save time, get more responses, give your visitors a better experience – all without lifting a finger.', 'jetpack' ),
+			'options'           => array(
+				array(
+					'type'      => 'anti-spam',
+					'slug'      => 'jetpack-anti-spam',
+					'key'       => 'jetpack_anti_spam',
+					'name'      => __( 'Anti-Spam', 'jetpack' ),
+				),
+			),
+			'default_option'    => 'anti-spam',
+			'included_in_plans' => array( 'anti-spam-plan' ),
```

Comment:
Is this correct? Is Jetpack Anti-Spam included in any Jetpack plans?

Reply:
Looking at the code and testing, it seems this needs to include every paid plan class: `personal-plan`, `premium-plan`, etc.